actions/helm_stage.go

```diff
@@ -94,15 +95,15 @@ func stageFiles(fs fileSys, ghFiles []ghFile, stagingDir string) {
 		if _, err := io.Copy(f, readCloser); err != nil {
 			log.Fatalf("Error writing contents to file %s (%s)", localFilePath, err)
 		}
-		log.Printf("File %s staged in '%s'", ghFile.FileName, stagingDir)
+		fmt.Printf("File %s staged in '%s'", ghFile.FileName, stagingDir)
 		defer readCloser.Close()
 		defer f.Close()
 	}
 }
 
 func updateFilesWithRelease(fp filePath, fs fileSys, release releaseName, walkPath string) error {
 	if release.Full == "" || release.Short == "" {
-		log.Printf("WORKFLOW_RELEASE (%s) and/or WORKFLOW_RELEASE_SHORT (%s) not provided so not amending staged files.", release.Full, release.Short)
+		fmt.Printf("WORKFLOW_RELEASE (%s) and/or WORKFLOW_RELEASE_SHORT (%s) not provided so not amending staged files.", release.Full, release.Short)
```

Comment:
also applies to the below `fmt.Printf`s that don't have trailing `\n` characters...